gui/elements/qt/object.py

```diff
@@ -51,6 +51,10 @@ def y(self) -> int:
     def width(self) -> int:
         return int(self.bounds.width)
 
+    @allure.step('Get title {0}')
```

Comment:
Probably you mean get attribute here, not just title?

Reply:
yes, it is a copypaste leftover, fixed